RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release
+
+The kubernetes-base-addons repository is intended to support a minimal base set of Addons required to provide monitoring, logging, alerting, and backups for every version of Kubernetes actively supported by the upstream Kubernetes community.
+It is not intended to be tied to any specific version of Kubernetes installer (ie. Konvoy or Kommander), or Kubeaddons.
+Changes to Konvoy or Kommander to create a resource needed by an Addon should be avoided.
+
+- [Schedule](#schedule)
+- [Considerations](#considerations)
+  - [Kubernetes](#kubernetes)
+  - [Branches and Tags](#branches-and-tags)
+- [Process](#process)
+  - [Testing Release (Weekly, Thursday)](#testing-release-weekly-thursday)
+  - [Stable Release (Weekly, Wednesday)](#stable-release-weekly-wednesday)
+
+## Schedule
+
+Releases should be weekly on Wednesdays, or as needed to address CVEs.
+
+## Considerations
+
+### Kubernetes
+
+The Addons in this repo are the minimum base set of supported<sup>[1](#footnote1)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+
+Kubernetes support must be maintained for the latest general release of Kubernetes and the two prior minor releases.
+At the time of this writing, the latest Kubernetes release is 1.17.2.
+Support from this repo, at this time, should cover 1.15.x, 1.16.x, and 1.17.x.
+
+### Branches and Tags
```

Comment:
Hey @joejulian what are your thoughts on having branches associated with konvoy releases? 

For example, say we have a `konvoy-1.3` branch that has all the commits up to when we GA'd 1.3. Now, for a `1.3.1` release, we can cherry-pick any commits from master that we want to include (eg security patches/bug fixes). 

This would make it easier to isolate (and track) which changes we want for a given version of Konvoy and avoid updating _all_ of the addons in the event we don't want to. Of course you can still cut tags/releases as `stable-1.15-xx` and `stable-1.16-xx` (which Konvoy would specify in its cluster.yaml configVersion) but this would simplify Konvoy releases a little more. WDYT?

Reply:
I think this creates a lot of work for developers and provides very little to no value for the customer. As long as we maintain a set of addons that successfully supports all of the kubernetes versions we support, there should be no need for a branch.

Reply:
Okay got it. Thinking it over, it might be more sensible to mark the Konvoy release as 1.4.0 (or 1.4-beta) since using a newer base-addons release/tag may include new features as well as bug fixes compared to its 1.3 version. Thanks, talking this out cleared up some misunderstandings on my end.